docs/HowToUsePeripheral.md

```diff
@@ -246,3 +251,89 @@ static void spi_select(struct spi_dev_s *dev, enum spi_dev_e devid, bool selecte
+RNUM:	int32 - Region Number
+RBASE:	int32 - Region Base Address
+RSIZE:	int32 - Region Size
+RATTR:	int32 - Region Attirbute
+-----------------------------------
+```
+
+```
+-----------------------------------------------------------------------------------------------------------
+                  MPU Init or Memory Region Setup Functions with Description
+-----------------------------------------------------------------------------------------------------------
+mpu_priv_stronglyordered() : Configure a region for privileged, strongly ordered memory
+mpu_priv_noncache()        : Configure a non-cacheable region as privileged internal memory
+mpu_priv_flash()           : Configure a region for privileged program flash
+mpu_priv_intsram()         : Configure a region as privileged internal SRAM region
+mpu_priv_extsram()         : Configure a region as privileged external SRAM region
+mpu_priv_intsram_wb()      : Configure a region as privileged internal SRAM region with WB/WA cache policy
+mpu_peripheral()           : Configure a region as privileged periperal address space
+mpu_user_flash()           : Configure a region for user program flash
+mpu_user_intsram()         : Configure a region as user internal SRAM
+mpu_user_extsram()         : Configure a region as user external SRAM
+mpu_user_intsram_wb()      : Configure a region as user internal SRAM with WB/WA cache policy
+mpu_control()              : Enable (or disable) the MPU
+-----------------------------------------------------------------------------------------------------------
+```
+
+The functions with name ```_priv_``` are used to configure the _kernel_(Privileged) MPU regions whereas the ```_user_``` functions are called to configure the _user_(Un-Privileged) MPU regions.
+
+MPU region setup related macros and functions are defined in **[mpu.h](../os/arch/arm/src/armv7-r/mpu.h)** under *os/arch/arm/src/armv7-r/* directory.
+
+### Example:
+```
+#ifdef CONFIG_ARMV7M_MPU
+int s5j_mpu_initialize(void)
+{
+#ifdef CONFIG_ARCH_CHIP_S5JT200
+	/*
+	 * Vector Table		0x02020000	0x02020FFF	4
+	 * Reserved		0x02021000	0x020217FF	2
+	 * BL1			0x02021800	0x020237FF	8
+	 * TizenRT		0x02023800	0x0210FFFF	946(WBWA)
+	 * WIFI			0x02110000	0x0215FFFF	320(NCNB)
+	 */
+
+	/* Region 0, Set read only for memory area */
+	mpu_priv_flash(0x0, 0x80000000);
+
+	/* Region 1, for ISRAM(0x02020000++1280KB, RW-WBWA */
+	mpu_user_intsram_wb(S5J_IRAM_PADDR, S5J_IRAM_SIZE);
+
+	/* Region 2, wifi driver needs non-$(0x02110000++320KB, RW-NCNB */
+	mpu_priv_noncache(S5J_IRAM_PADDR + ((4 + 2 + 8 + 946) * 1024), (320 * 1024));
+
+	/* Region 3, for FLASH area, default to set WBWA */
+	mpu_user_intsram_wb(S5J_FLASH_PADDR, S5J_FLASH_SIZE);
+
+	/* region 4, for Sflash Mirror area to be read only */
+	mpu_priv_flash(S5J_FLASH_MIRROR_PADDR, S5J_FLASH_MIRROR_SIZE);
+
+	/* Region 5, for SFR area read/write, strongly-ordered */
+	mpu_priv_stronglyordered(S5J_PERIPHERAL_PADDR, S5J_PERIPHERAL_SIZE);
```

Comment:
Address is the periperal's, but this uses stronglyordered, not mpu_periperal. Because this is a porting guide, we had better say what difference between them, when we can use it and so on.